### ISSUE TITLE
Black screen after 10min

### ISSUE THREAD

**Issue by haymaan** [roramirez/MMM-ImagesPhotos#10]
I've installed the Imagephotos and its work fine,but the screen goes black after 10min,i remove the modules the mirror works fine ,i've install it again and the same probleem screen goes black...please help

**Comment by kprasann1979** [comment]
Yes, it works , but after few minutes goes blank...am using this inside the MMM-carousel module though if thats causing any problem...? All other modules in the carousel works fine...

{
                        module: "MMM-Carousel",
                        config: {
                                transitionInterval: 30000,
                                ignoreModules: ['alert','updatenotification','MMM-Remote-Control','worldclock','MMM-cryptocurrency','newsfeed'],
                                mode: 'slides',
                                slides: ['currentweather', 'weatherforecast','MMM-Globe','MMM-rfacts','MMM-SystemStats','MMM-ImagePhotos']                $
                        }
                },



 {
                        module: "MMM-ImagesPhotos",
                        position: 'top_right',
                        config: {
                                opacity: 0.9,
                                animationSpeed: 2000,
                                maxWidth: "250px",
                                maxHeight: "250px",
                                getInterval: 2000,
                                updateInterval: 2000
                        }
                },

**Comment by haymaan** [comment]
no one can help ... ?

**Comment by roramirez** [comment]
Can you tell us the version of MagicMirror installed in your system and how many photos are you displaying (size, numbers, etc..)

**Comment by ThatsFluke** [comment]
uhh, its most likely because of screen blanking; a feature built-into Raspbian.
